bob-common/src/mapper.rs

```diff
@@ -291,23 +291,26 @@ impl Virtual {
             .collect()
     }
 
-    pub fn get_operation(&self, key: BobKey) -> (VDiskId, Option<DiskPath>) {
+    pub fn get_operation(&self, key: BobKey) -> (VDiskId, Option<Vec<DiskPath>>) {
```

Comment:
Please, use `SmallVec<[DiskPath; 1]>` here instead of `Vec<DiskPath>`. `SmallVec` stores up to 1 element on the stack, and in 99% of cases we will have just one element